beps/0005-split-backend-discovery/README.md

```diff
@@ -186,13 +220,48 @@ paths:
             application/json:
               schema:
                 type: object
+                # using an object here so that we can add extra metadata like cache ttl, etc.
                 properties:
-                  internal:
+                  url:
                     type: string
-                  external:
+  /by-plugin/{pluginId}/external-base-url:
+    get:
+      summary: Get external registration URL for a given pluginId.
+      operationId: getExternalRegistrationByPlugin
+      parameters:
+        - name: pluginId
+          in: path
+          description: The plugin ID to get information for.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  url:
```

Comment:
Are we delegating entirely to the gateway to maintain health checks and so on? Was wondering if you'd thought about returning more stuff like the actual health check URL + region etc here etc so that clients can make their own choices.

Reply:
That was my thinking. Maybe similar to my above comment, I wanted the gateway to handle a lot of this and fit this design narrowly into the existing discovery API. If we want to adjust this for better support for client based routing, I'll need to take another look at this.